app/validators/defendant_validator.rb

```diff
@@ -19,9 +19,9 @@ def validate_claim
   end
 
   def validate_date_of_birth
-    validate_presence(:date_of_birth, 'blank')
-    validate_on_or_before(10.years.ago, :date_of_birth, 'check')
-    validate_on_or_after(120.years.ago, :date_of_birth, 'check')
+    validate_presence(:date_of_birth, :blank)
```

Comment:
This modified file relates to the `defendants` form_step, is this required for `supporting_evidence`?